docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,151 @@
+# Tizen Window System Shell
+
+
+The Tizen window system shell (TZSH) is an interface for manipulating windows of the sytem graphical user interfaces (GUI) services such as the quickpanel and the softkey. It provides C libraries that allow developers to control windows of system GUI services and be notified state changes of them within an application.
+
+Since the system GUI services are different processes from the application, all of operations of the TZSH are performed using the inter-process communication (IPC) through the display server. Below figure shows a simple S/W architecture of the TZSH.
+
+**Figure: Layer diagram for the Tizen window system shell**
+
+![Layer diagram for the Tizen window system shell](./media/tzsh_arch.png)
+
+The TZSH provides following libraries to support each system GUI services in the Tizen:
```

Comment:
To support each system GUI service in Tizen, the TZSH provides the following libraries:

Reply:
Thanks I will change it as your comment.